book/doctrine.rst

```diff
@@ -1204,7 +1204,8 @@ to the given ``Category`` object via their ``category_id`` value.
         $category = $product->getCategory();
 
         // prints "Proxies\AppBundleEntityCategoryProxy"
-        echo get_class($category);
+        dump(get_class($category));
+        die();
```

Comment:
From the original PR https://github.com/symfony/symfony-docs/pull/5285

`In the framework, dump() needs to be used with a die(), else it won't show up (it'll show up in the WDT).`